src/js/contentscripts/socialwidgets.js

```diff
@@ -330,7 +330,7 @@ function replaceIndividualButton(tracker) {
 *                            replaced
 */
 function getTrackerData(callback) {
-  chrome.runtime.sendMessage({checkReplaceButton:document.location}, function(response) {
+  chrome.runtime.sendMessage({checkReplaceButton:document.location.hostname}, function(response) {
```

Comment:
According to https://wiki.mozilla.org/RapidRelease/Calendar, Firefox 55 (current Beta) lands on August 8th, so we need to have a release before then (or Firefox needs to revert their breaking change).